### PR TITLE
Add pymunk

### PR DESCRIPTION
**Pull request by viblo**

### CODE REVIEW

**Comment by viblo** [comment]
Took some trial and error, but should be ready now :)

File: recipes/pymunk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+build:
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: 
+    - export LDFLAGS="-dynamiclib $LDFLAGS"  # [osx]
+    - export LD=clang  # [osx]
+    - export CFLAGS="$CFLAGS -O3"  # [linux]
+    - export LDFLAGS="-shared $LDFLAGS"  # [linux]
+    - export LD=gcc  # [linux]
```
Comment (beckermr) [inline recipes/pymunk/meta.yaml]: These lines are ok for toolchain builds but will break the migration to the anaconda compilers.

File: recipes/pymunk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - python -m pymunk.tests UnitTest
+
+about:
+  home: https://www.pymunk.org
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Pymunk is a easy-to-use pythonic 2d physics library'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (beckermr) [inline recipes/pymunk/meta.yaml]: Remove this comment.

**Comment by viblo** [comment]
Thanks for the review. I have removed the comment. The compiler stuff I will take when it comes.

File: recipes/pymunk/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/pymunk/meta.yaml]: You missed the comment to remove. :)

**Comment by viblo** [comment]
Sorry, my mistake. Removed the wrong comment. (I assume the comment about no-deps can/should stay since its useful?)

**Comment by beckermr** [comment]
Yep. Looks good! Once the tests pass we can merge! Thanks for the contribution!

**Pull request merged**